GitDepend/Helpers/VersionUpdateHelper.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+using RestSharp;
+
+namespace GitDepend.Helpers
+{
+    /// <summary>
+    /// This class will check to see if there is a new version of gitdepend available from the github api.
+    /// </summary>
+    internal static class VersionUpdateHelper
```

Comment:
Rather than making this VersionUpdateHelper, let's call it VersionUpdateChecker, and have it implement IVersionUpdateChecker, and hook that up in Unity. This will make it mockable.